src/zones/cc-zone.js

```diff
@@ -100,12 +100,16 @@ export class CcZone extends LitElement {
 
   static get styles () {
     return [
-      defaultThemeStyles,
       skeletonStyles,
       // language=CSS
       css`
         :host {
           --lh: 1.5em;
+          /*
+          * This allows cc-pricing-header to override the text color of the cc-zone (title and tags) on hover.
+          * It is necessary because we don't have easy access to the cc-zone host since it is used inside a slot in the shoelace select.
+          */
```

Comment:
I checked and this can be done without it. We'll discuss this but slotted elements are not in the shadow DOM and are accessible to CSS.

Reply:
I think my explanation is just completely missing the point (I wrote it 3 weeks after actually changing this, which was not very clever).

Could we discuss this in sync ? :wink: